seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -39,6 +39,12 @@
         "El número menor no siempre será el primer elemento en el arreglo.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
       ],
+      "namePtBR": "Some Todos os Números em um Intervalo",
+      "descriptionPtBR": [
```

Comment:
Sorry I didn't see this earlier. I had a long discussion with several Brazilian campers, and I think we absolutely should support Brazilian Portuguese. Note that we are also supporting both traditional (zh-tw) and simplified Chinese (zh-cn) so there is a need for multi-character 'postfixes'.